symforce/codegen/codegen_util.py

```diff
@@ -574,6 +581,25 @@ def load_generated_package(name: str, path: T.Openable) -> T.Any:
     return module
 
 
+def load_generated_function(func_name: str, path_to_package: T.Openable) -> T.Any:
+    """
+    Loads the function with name func_name found inside the package located at
+    path_to_package.
+
+    Preconditions:
+        path_to_package is a python package with an `__init__.py` containing a module
+            defined in `func_name.py` which in turn defines an attribute named `func_name`.
+            Note: the precondition will be satisfied if the package was generated by
+            `Codegen.generate_function` from a `Codegen` function with name `func_name`.
```

Comment:
I think it would be helpful to have a brief example here of what results from `Codegen.generate_function` you should pass here.  It's also maybe not 100% obvious that this returns the function, probably easy to figure out but could annotate the return type as `T.Callable` or mention in the docstring that the return value is the funcion

Reply:
Good ideas.